ntc_templates/templates/alcatel_aos_show_vlan_port.textfsm

```diff
@@ -0,0 +1,12 @@
+Value VLAN_ID (\d+)
+Value PORT ((\d+\/)?\d+\/\d+)
+Value TYPE (default|qtagged)
+Value STATUS (forwarding|inactive)
```

Comment:
```suggestion
Value TYPE (\S+)
Value STATUS (\S+)
```

Probably better to make the pattern more flexible for the long term.
In my testing using this pattern causes no ill effects with the test data.

Reply:
added suggestion! :)